packages/StatusTracker/src/components/Point/Point.styles.js

```diff
@@ -6,18 +6,33 @@ import { fontSize, lineHeight, spacer } from "@paprika/stylers/lib/helpers";
 import * as types from "../../types";
 
 const passedPath = css`
-  border-bottom: 2px solid ${tokens.color.greenLighten10};
+  border-bottom: 2px solid ${tokens.color.black};
 `;
 
 const futurePath = css`
   border-bottom: 2px dashed ${tokens.color.blackLighten20};
 `;
 
+const pathWidth = css`
+  max-width: ${spacer(8)};
+`;
+
+const firstChildPathWidth = css`
+  max-width: ${spacer(12)};
+`;
+
+const currentPathWidth = css`
+  max-width: ${spacer(16)};
+`;
+
 export const PointWrapper = styled.li`
   align-items: center;
   display: inline-flex;
   flex-grow: 1;
   list-style: none;
+  max-width: ${({ hasOverflowMenu }) => (hasOverflowMenu !== undefined ? spacer(18) : "")} !important;
```

Comment:
I'm not super pro with styled-components yet, if someone has a better alternative than this would be great.

`hasOverFlowMenu => space(18)`
`if CURRENT => space(16), otherwise space(8)`

Reply:
Why do we need to check if it's undefined?

Reply:
We only want to apply this styling if we use the overflow menu button. The undefined values get passed if the overflow menu is not there. So if it is not undefined then that means we use the overflow menu button